native/phash.cc

```diff
@@ -0,0 +1,23 @@
+#ifndef PHASH_H_
+#define PHASH_H_
```

Comment:
The guards should not appear in `.cc` files.

Reply:
We had it in `utils.cc`. However, you're right.